package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -87,12 +103,18 @@ export interface SpotlightTourProviderProps {
   steps: TourStep[];
 }
 
+const DEFAULT_FLOATING_PROPS: FloatingProps = {
+  middleware: [flip(), offset(4), shift()],
+  placement: "bottom",
+};
+
 /**
  * React provider component to get access to the SpotlightTour context.
  */
 export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProviderProps>((props, ref) => {
   const {
     children,
+    floatingProps: floatingMiddlewareProps = DEFAULT_FLOATING_PROPS,
```

Comment:
We don't need to rename this prop (maybe it's just a leftover of the refactor):
```suggestion
    floatingProps = DEFAULT_FLOATING_PROPS,
```

Reply:
This is the object i meant in the previous comment, we have this object coming from the provider and we cant use the same name for the context

Reply:
we can change this one into floatingProps with no problem but the ```floatingUiConfigurations``` should have a different name than ```floatingProps```